lib/listcoef.gd

```diff
@@ -101,26 +101,53 @@ DeclareOperation(
 #O  MultRowVector( <list>, <mul> )
 ##
 ##  <#GAPDoc Label="MultRowVector">
+##  <#GAPDoc Label="MultRowVectorLeft">
+##  <#GAPDoc Label="MultRowVectorRight">
 ##  <ManSection>
 ##  <Oper Name="MultRowVector" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorLeft" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorRight" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Returns>nothing</Returns>
 ##
 ##  <Description>
-##  The five argument version of this operation replaces
+##  Note that <C>MultRowVector</C> is only a Synonym for
+##  <C>MultRowVectorLeft</C>.
```

Comment:
"only" -> "just", "Synonym" -> "synonym"

It also seems weird to start with this side remark. I'd move this sentence to a later spot, say right after `MultRowVectorLeft` has been declared.